flask_ckeditor/utils.py

```diff
@@ -15,3 +20,20 @@ def random_filename(old_filename):
     ext = os.path.splitext(old_filename)[1]
     new_filename = uuid.uuid4().hex + ext
     return new_filename
+
+
+def cleanify(text, *, allow_tags=None):
+    """clean the input from client, this function rely on bleach,
+
+
+    Args:
```

Comment:
Use the docstring style that matches the [existing code](https://github.com/helloflask/flask-ckeditor/blob/master/flask_ckeditor/__init__.py#L20C7-L27).